src/app/(app)/funds/actions/add-funds.ts

```diff
@@ -5,9 +5,12 @@ import { Tables } from "@/lib/db";
 import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
 import { FundSchemaType } from "../components/FundFormBuilder";
 
-export const createFunds = async (values: FundSchemaType) => {
+export const addFunds = async (values: FundSchemaType) => {
   const supabase = await getSupabaseClient();
-  const { error } = await supabase.from(Tables.Funds).insert(values);
+  const { error } = await supabase.from(Tables.Funds).insert({
+    member_id: values.memberId,
+    amount: values.amount,
```

Comment:
I would like to know what happens when data is inserted successfully?

Reply:
@Mubashar1009 Here, an error is thrown if the data is not successfully inserted; otherwise, it revalidates the path.